storage-market.md

```diff
@@ -115,16 +80,14 @@ TODO: 'complete' isnt really the right word here, as it implies that the deal is
 
 ## The Power Table
 
-The `power table` is exported by the storage market for use by consensus. There isn't actually a concrete object that is the power table, instead, there is a 'total power' exported by the storage market actor, and each individual miner reports its power through their actor.
+The storage market actor maintains a total committed storage amount, aka "total power". The "power table" is a logical construct which is the fraction of this total power that each miner contributes. This is used by the consensus algorithm.
 
-TODO: rephrase the above to make it clear that power is updated only on PoSt submission and when slashed.
+Each individual miner reports its power through its on-chain actor. A miner's power (and hence the total power) is updated only upon PoSt submission (when the set of sectors being proved may change) or when a miner is slashed.
```

Comment:
Might be good to note that this submission is dependent on the individual miner, rather than a global operation

Reply:
Actually this is already clear, as this is talking about a specific miner, nvm